runtime/plugin/extra-keymaps.vim

```diff
@@ -0,0 +1,43 @@
+" Extra default keymaps and keyboard behavior
+
+" Exit quickly if
+" 1) the user has disabled the plugin
+" 2) the file is already loaded
+if exists('g:nvim#defaults#maps#use') && g:nvim#defaults#maps#use == 0
+  || exists('g:nvim#defaults#maps#loaded') && g:nvim#defaults#maps#loaded == 1
+  finish
+endif
+
+" Set defaults:
+" The user can disable parts of the plugin separatedly
+" by setting any of these to 0
+if !exists("g:nvim#defaults#maps#leave_term_mode")
+  let g:nvim#defaults#maps#leave_term_mode = 1
+endif
+if !exists("g:nvim#defaults#maps#Y_eol")
```

Comment:
The set of mappings is quite varied; I don't think wanting to disable the whole of it is more likely than wanting to disable any of these separately.